src/main/java/com/afj/solution/buyitapp/controller/api/v1/UserController.java

```diff
@@ -52,4 +61,20 @@ UserResponse me() {
         return userService.getMe(userId);
     }
 
+    @ApiOperation(value = "Create New User", authorizations = {@Authorization("Bearer")})
+    @ApiResponses({
+            @ApiResponse(code = 201, message = "User created"),
+            @ApiResponse(code = 409, message = "User Already exists"),
```

Comment:
`409` it is correct?

Reply:
![image](https://user-images.githubusercontent.com/37942384/182106839-9029d045-dfa2-4e3d-921f-da8397be15d8.png)